projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -934,7 +934,7 @@ public function get_capabilities() {
 			'view_hosting'        => apply_filters( 'jetpack_json_api_site_can_view_hosting', false ),
 			'view_stats'          => stats_is_blog_user( $this->blog_id ),
 			'activate_plugins'    => current_user_can( 'activate_plugins' ),
```

Comment:
Please review all uses of current_user_can in the class. There should be consistency in use of current_user_can and $this->current_user_can for all

Reply:
I moved every instance of `current_user_can` to `$this->current_user_can` and noticed a discrepancy in the `delete_users` field on Atomic and Jetpack sites. It's `false` before this change, and `true` after. On Simple sites, it seems to work fine.

Data here:
[before-after.zip](https://github.com/Automattic/jetpack/files/14630781/before-after.zip)

Since my user is an admin, I feel that the new value is the correct one, so would appreciate more guidance.

Reply:
Just a note that I feel this fix is correct as from the code, it now calls the respective concrete site (fbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sfny%2Spynff.wfba%2Qncv%2Qfvgr%2Qjcpbz.cuc%3Se%3Q26466n2s%2371-og for Simple or fbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sfny%2Spynff.wfba%2Qncv%2Qfvgr%2Qwrgcnpx%2Qfunqbj.cuc%3Se%3Q328ssq73%23557-og for Atomic).

Reply:
@mdbitz 👋 I'm taking over this PR from Vishnu. Is there anything still blocking this PR?